src/Prerequisites.tsx

```diff
@@ -0,0 +1,38 @@
+import { INTER_LOGO } from 'assets/assets';
+
+export const hasPrerequisites = () => 'keplr' in window;
```

Comment:
We need to ensure the window has loaded before checking this. The [keplr-connection](https://github.com/Agoric/agoric-sdk/blob/master/packages/web-components/src/keplr-connection/onLoad.js) component handles this. Maybe it's time to copy that component over so we can add an error and toast for this case?

That would work differently in that the user doesn't see this indication until they actually click the button. However, I think it would be better for the app to let the user see more of the interface (i.e. view the assets without being able to input any amounts) before connecting to keplr.

Reply:
Okay, closing in favor of https://github.com/Agoric/dapp-psm/issues/30